exalearn-api/ExaLearn.WebApi/Startup.cs

```diff
@@ -65,7 +65,8 @@ public void ConfigureServices(IServiceCollection services)
                 });
             });
 
-            services.AddDbContext<ExaLearnDbContext>(options => options.UseNpgsql(Configuration.GetConnectionString("DbContext")));
+            services.AddDbContext<ExaLearnDbContext>(options => options.UseNpgsql(Configuration.GetConnectionString("DbContext"), 
+                b => b.MigrationsAssembly("ExaLearn.WebApi")));
```

Comment:
for what

Reply:
> for what

without it migration cant build

Reply:
strange

Reply:
it may look like a local conflict of migration. did you try to remove your old migrations from the project and restart migrations script once again